lab3/main.py

```diff
@@ -0,0 +1,71 @@
+
+def max_equal(arr, n):
```

Comment:
parameter "n" is not used, please remove it.
 also provide the full name of parameter "arr"